lib/notification_service/notification_service.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'package:firebase_messaging/firebase_messaging.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+
```

Comment:
Before starting each function, add some comments regarding what you have done inside that function.